src/main/java/org/folio/consortia/client/PermissionsClient.java

```diff
@@ -21,4 +22,7 @@ public interface PermissionsClient {
 
   @PostMapping(value = "/{userId}/permissions?indexField=userId", consumes = MediaType.APPLICATION_JSON_VALUE)
   void addPermission(@PathVariable("userId") String userId, Permission permission);
+
+  @DeleteMapping(value = "/perms/users/{id}")
```

Comment:
I am not sure that this endpoint deletes by userId, it seams that this param - it is id of permision user record, not user id

Reply:
https://github.com/folio-org/mod-permissions/blob/master/src/main/java/org/folio/rest/impl/PermsAPI.java#L341
If we looking at implementation, seems just id used , not userId, but need to check maybe locally